sql/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -307,6 +310,10 @@ private void fillLocationsFromShardIterators(Map<String, Map<String, List<Intege
         return primaryKeys;
     }
 
+    public List<ColumnIdent> getNotNullColumns() {
```

Comment:
I think the information if a column is nullable or not should be part of the Reference and not a separate info on the TableInfo